apps/artemis-web/pages/hijacks.tsx

```diff
@@ -28,34 +28,74 @@ const useStyles = makeStyles((theme) => ({
 
 const HijacksPage = (props) => {
   const [isLive, setIsLive] = useState(true);
+  const isDevelopment = process.env.NODE_ENV === 'development';
+  const isBrowser = typeof window !== 'undefined';
 
-  if (process.env.NODE_ENV === 'development') {
-    if (typeof window !== 'undefined') {
-      // eslint-disable-next-line @typescript-eslint/no-var-requires
-      const { worker } = require('../utils/mock-sw/browser');
-      worker.start();
-    }
+  if (isDevelopment && isBrowser) {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const { worker } = require('../utils/mock-sw/browser');
+    worker.start();
   }
 
   const classes = useStyles();
 
-  const [filter, setFilter] = useState(0);
+  const [filterDate, setFilterDate] = useState(0);
+  const [filterStatus, setFilterStatus] = useState('');
   const [filterButton, setFilterButton] = useState(0);
   const [distinctValues, setDistinctValues] = useState([]);
   const [selectState, setSelectState] = useState('');
+  const [statusButton, setStatusButton] = useState(-1);
+
+  const statuses = [
+    'Ongoing',
+    'Dormant',
+    'Resolved',
+    'Ignored',
+    'Under Mitigation',
+    'Withdrawn',
+    'Outdated',
+  ];
 
   const user = props.user;
 
+  const findStatus = (row) => {
+    if (row.withdrawn) return 'Withdrawn';
+    else if (row.resolved) return 'Resolved';
+    else if (row.ignored) return 'Ignored';
+    else if (row.active) return 'Active';
+    else if (row.dormant) return 'Dormant';
+    else if (row.under_mitigation) return 'Under Mitigation';
+    else if (row.outdated) return 'Outdated';
+    else return '';
+  };
+
+  const setStatus = (id) => {
```

Comment:
why pass an ID and not have the name directly? The statuses list does not make sense.